scholia/arxiv.py

```diff
@@ -16,17 +16,14 @@
 from __future__ import absolute_import, division, print_function
 
 import json
-
 import os
-
 import re
 
 import sys
 
 import requests
-
 from feedparser import parse as parse_api
-
+from flask import current_app
```

Comment:
Last time you flagged this as a problem https://github.com/WDscholia/scholia/pull/2340#discussion_r1377858509

Can be removed as it only seems like it is for debugging